### PR TITLE
CIV-11547 - Claim Issue: Issue Work Allocation

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-11547

### Change description ###
Added WA Task to trigger WA task for Claim Issue journey when Claim Issue Language = 'English + Welsh'
 

WA Task 

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by jarekPierchala — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Review by manish14836 — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskCompletionTest.java [hunk 1]
```diff
@@ -286,6 +286,6 @@ void if_this_test_fails_needs_updating_with_your_changes() {
 
         //The purpose of this test is to prevent adding new rows without being tested
         DmnDecisionTableImpl logic = (DmnDecisionTableImpl) decision.getDecisionLogic();
-        assertThat(logic.getRules().size(), is(32));
+        assertThat(logic.getRules().size(), is(33));
```
Comment (paul-pearson) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskCompletionTest.java:289]: Check if tests for the new rules are needed as well.

**Review by paul-pearson — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaPermissionTest.java [hunk 2]
```diff
@@ -747,4 +747,28 @@ void given_hmc_taskType_when_evaluate_dmn_then_it_returns_expected_rule(String t
             )
         )));
     }
+
+    @ParameterizedTest
+    @CsvSource(value = {
+        "claimantWelshRequest"
```
Comment (manish14836) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaPermissionTest.java:753]: Doesn't need to be parameterized if only 1 value is input

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskCompletionTest.java [hunk 3]
```diff
@@ -221,6 +221,25 @@ static Stream<Arguments> scenarioProviderTDoRR() {
         );
     }
 
+    static Stream<Arguments> scenarioProviderUpTD() {
+
```
Comment (manish14836) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskCompletionTest.java:225]: Remove this empty line

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskCompletionTest.java [hunk 4]
```diff
@@ -281,11 +300,21 @@ void given_event_ids_should_evaluate_decOnReq_dmn(String eventId, List<Map<Strin
         MatcherAssert.assertThat(dmnDecisionTableResult.getResultList(), is(expectation));
     }
 
+    @ParameterizedTest(name = "event id: {0}")
+    @MethodSource({"scenarioProviderUpTD"})
+    void given_event_ids_should_evaluate_UploadTransDoc_dmn(String eventId, List<Map<String, String>> expectation) {
+
```
Comment (manish14836) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskCompletionTest.java:306]: Remove this empty line

**Comment by neeta-hmcts** [comment]
closing this PR and opened new PR https://github.com/hmcts/civil-wa-task-configuration/pull/489

**Pull request closed without merging**